nexus/social_media/models.py

```diff
@@ -0,0 +1,34 @@
+# Third Party Stuff
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+# Nexus Stuff
+from nexus.base.models import TimeStampedUUIDModel, ImageMixin
+
+
+class Post(TimeStampedUUIDModel, ImageMixin):
+    """ Post model class to provide post related field
+    """
+    posted_by = models.ForeignKey('users.User', on_delete=models.CASCADE, verbose_name=_('Posted by'))
+    posted_at = models.CharField(
+        _('Posted at platform'),
+        choices=(('fb', 'Facebook'),
+                 ('twitter', 'Twitter'),
+                 ('linkedin', 'Linkedin')),
+        max_length=10)
+    scheduled_time = models.DateTimeField(_('Scheduled at'), null=True, blank=True)
+    approval_time = models.DateTimeField(_('Approved at'), null=True, blank=True)
+    posted_time = models.DateTimeField(_('Posted at'), null=True, blank=True)
+    text = models.TextField(_('Content Text'), blank=True, null=True)
+    is_approved = models.BooleanField(
+        _('Is post approved'), default=False,
+        help_text='is the post approved by Nexus administrator?')
+    is_posted = models.BooleanField(
+        _('Is posted'), default=False,
+        help_text='is the post published?')
+
+    class Meta:
```

Comment:
Include `verbose_name`, `verbose_name_plural` and `db_table` attributes in Meta.

Also, please follow rules mentioned in `coding_rules.md`

Reply:
ACK